ngraph_bridge/ngraph_builder.cc

```diff
@@ -2893,20 +2878,13 @@ Status Builder::TranslateGraph(
   //
   {
     ngraph::pass::Manager passes;
-    ngraph::pass::PassConfig pass_config;
-    // set/honor the defaults, unless specified via env var
-    auto set_default = [&pass_config](std::string pass, bool enable) {
-      auto enables_map = pass_config.get_enables();
-      if (enables_map.find(pass) == enables_map.end())
-        pass_config.set_pass_enable(pass, enable);
-    };
-    set_default("ConstantFolding", false);
-    set_default("TransposeSinking", true);
-    if (pass_config.get_pass_enable("ConstantFolding"))
+    passes.register_pass<pass::TransposeSinking>();
```

Comment:
Should the pass be registered even if the user does not want to run it?

Reply:
Done.